CONTRIBUTION.md

```diff
@@ -53,42 +41,96 @@ The format can be described more formally as follows:
 <BLANK LINE>
 <why this change was made>
 <BLANK LINE>
-<footer>
+<Footer>
 ```
-Example for a sample feature:
+
+A real world example would look like
+
 ```
-[docs] Add the Guidlines
+[docs] Add the Guidelines
 
-Cupcake ipsum dolor sit. Amet tart cheesecake tiramisu chocolate cake topping.
-Icing ice cream sweet roll. Biscuit dragée toffee wypas.
+Contribution guidelines out of date and no longer reflected current practices.
```

Comment:
This is just a statement. What are you doing to fix it? IOW, the example should follow the guidelines.

Reply:
```suggestion
The contribution guidelines were not aligned with current practices. Update the
guidelines and reorganize it to bring it up to date.
```